cmd/mc/skin/add.go

```diff
@@ -47,7 +47,7 @@ func newAddCmd(app *cli.App, account string) *cobra.Command {
 
 	o.account = account
 
-	cmd.Flags().StringVar(&o.variant, "variant", "classic", "Skin variant [classic/slim]")
+	cmd.Flags().StringVar(&o.variant, "variant", "", "Skin variant [classic/slim] (defaults to classic)")
```

Comment:
Seems like a bad change, why not let cobra document it?

Reply:
I default it to blank because it allows for it to be inferred from the texture metadata if the passed in argument is a player uuid/username, i didn't want to disallow overriding the variant encase you pick a skin that is slim but the applied account is classic. If there's a better approach, then let me know